app/javascript/components/main/Root.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import PropTypes from "prop-types"
+import {BrowserRouter, Route, NavLink} from 'react-router-dom'
+import Banner from 'components/main/Banner'
+import Menu from 'components/main/Menu'
+
+class Root extends React.Component {
+    render() {
+        return (
+            <div>
```

Comment:
instead of just creating empty div you can use `<React.Fragment>` which will get you the same result just without the extra div.